src/components/NavBar.tsx

```diff
@@ -0,0 +1,60 @@
+import './NavBar.css'
+import ToggleLightMode from './ToggleLightMode'
+import userIcon from '../assets/user.svg'
+import logo from '../assets/cityLogo.svg'
+import { useState } from 'react'
+
+const NavBar = ({colorMode, mode, login} : {colorMode: any, mode: string, login: boolean}) =>{
+    const [colorRoute, setColorRoute] = useState(1)
+
+    function colorSwitcher(actualID: number, colorRoute: number): string{
+        if (actualID === colorRoute){
+            return "yellow"
+        }
+        return ""
+    }
+
+    return(
+        <>
+            <div id="nav-container">
+                <img src={logo} alt="" />
+
+                <div id='nav-pages'>
+                    <li className='Nav-route' onClick={() => setColorRoute(1)} style={{color: colorSwitcher(1, colorRoute)}}>
+                        <strong>Home</strong>
+                    </li>
+                    <li className='Nav-route' onClick={() => setColorRoute(2)} style={{color: colorSwitcher(2, colorRoute)}}>
+                        <strong>Exposições</strong>
+                    </li>
+                    <li className='Nav-route' onClick={() => setColorRoute(3)} style={{color: colorSwitcher(3, colorRoute)}}>
+                        <strong>Acervo</strong>
+                    </li>
+                    <li className='Nav-route' onClick={() => setColorRoute(4)} style={{color: colorSwitcher(4, colorRoute)}}>
+                        <strong>Editais e Normas</strong>
+                    </li>
+                </div>
```

Comment:
Utilizar Botão e Menu do Material UI, o menu abre quando o mouse entra no botão e fecha quando o mouse sai do botão ou dos itens do menu. Exemplo: menu acervo tem itens 'coleções', 'pesquisar item' e 'cadastrar item'. Utilizar estilos de acordo com o protótipo

[Discussão no Stack Overflow](https://stackoverflow.com/questions/55318477/how-to-make-material-ui-menu-based-on-hover-not-click)